content/docs/working-groups/meta/rfc-drafts/compiler-team-contributors.md

```diff
@@ -1,7 +1,7 @@
-- Feature Name: (fill me in with a unique ident, `my_awesome_feature`)
-- Start Date: (fill me in with today's date, YYYY-MM-DD)
-- RFC PR: [rust-lang/rfcs#0000](https://github.com/rust-lang/rfcs/pull/0000)
-- Rust Issue: [rust-lang/rust#0000](https://github.com/rust-lang/rust/issues/0000)
+Feature Name: (fill me in with a unique ident, `my_awesome_feature`)
+ Start Date: (fill me in with today's date, YYYY-MM-DD)
+ RFC PR: [rust-lang/rfcs#0000](https://github.com/rust-lang/rfcs/pull/0000)
+ Rust Issue: [rust-lang/rust#0000](https://github.com/rust-lang/rust/issues/0000)
```

Comment:
Maybe we should keep these as a list like actual RFCs:

![image](https://user-images.githubusercontent.com/1295100/60489381-1e60c380-9c9c-11e9-8df6-ce84f2398312.png)